src/java/htsjdk/samtools/cram/build/ContainerFactory.java

```diff
@@ -192,8 +179,15 @@ private static Slice buildSlice(final List<CramCompressionRecord> records,
         int prevAlStart = slice.alignmentStart;
         for (final CramCompressionRecord record : records) {
             record.alignmentDelta = record.alignmentStart - prevAlStart;
+            if (record.alignmentDelta <0) {
+                System.out.println("hgotcha");
+            }
             prevAlStart = record.alignmentStart;
-            writer.write(record);
+            try {
+                writer.write(record);
+            } catch (ArrayIndexOutOfBoundsException e) {
+                e.printStackTrace();
+            }
```

Comment:
We should remove this, or at the very least we shouldn't catch without re-throwing.